### PR TITLE
Add vote by mail tool, election reminders tool, and update docs

### PR DESCRIPTION
**Pull request by camdenlee**
This is dependent on changes in `public-assets` and `voteamerica`. Should be released today.

### CODE REVIEW

**Review by nickcatal — APPROVED** [review]

File: app/multi_tenant/templates/multi_tenant/manage/embed.html [hunk 1]
```diff
@@ -35,6 +35,19 @@ <h1>{{ user.active_client }} Embed Code</h1>
                 </div>
             </div>
         </div>
+        <div>
+            <div class="card">
+                <div class="card-header">
+                    <a href="https://www.voteamerica.com/vote-by-mail/">Vote By Mail Tool</a>
+                </div>
+                <div class="card-body">
```
Comment (benweissmann) [inline app/multi_tenant/templates/multi_tenant/manage/embed.html:43]: Can we add a note here that this is the same tool as Absentee but with a different header?

File: docs/turnout/embed/free.md [hunk 2]
```diff
@@ -30,13 +30,27 @@ Premium tool users: please use the customized embed code found by logging in at
 <div class="voteamerica-embed" data-subscriber="public" data-tool="absentee"></div>
 ```
 
+## Vote by mail tool
+
+```html
```
Comment (benweissmann) [inline docs/turnout/embed/free.md:35]: Ditto on adding a note that this is the same as the absentee tool

**Review by benweissmann — APPROVED** [review]
Looks good, we should just note that VBM and absentee tools are the same with different headers

**Comment by camdenlee** [comment]
fixed them both so going to merge!

**Pull request merged**